### DISCUSSION TITLE
Cannot pull private image, despite login succeeded

### DISCUSSION THREAD

**Discussion by ZhilinS** [community/community#24942] [Actions]
<p>Hi there.<br>
I have a private repo and an image is pushed through github actions. It does appear in packages (see attached pic)</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/c/6cc525e71f412149bf2eb90116f6db978dc1fba5.png" data-download-href="/uploads/short-url/fwdXbnPAdE29AROpF09jDv0nPOl.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/5133580/181087845-fb6b3eef-3c81-4919-bd5c-401d69ad9d59.png" alt="image" data-base62-sha1="fwdXbnPAdE29AROpF09jDv0nPOl" width="690" height="395" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cc525e71f412149bf2eb90116f6db978dc1fba5_2_690x395.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cc525e71f412149bf2eb90116f6db978dc1fba5_2_1035x592.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cc525e71f412149bf2eb90116f6db978dc1fba5_2_1380x790.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cc525e71f412149bf2eb90116f6db978dc1fba5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1652×948 73.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>After that I’m trying to do docker login with CR_PAT, which I saved on my local machine, it goes flawlessly.</p>
<pre><code class="lang-auto">docker login ghcr.io -u zhilins
Password:
Login Succeeded
</code></pre>
<p>Then I try to inspect an image and get this</p>
<pre><code class="lang-auto">docker inspect ghcr.io/zhilins/blog/app-ui:6afd39d23078e2a2558c32332e563db8345bef78
[]
Error: No such object: ghcr.io/zhilins/blog/app-ui:6afd39d23078e2a2558c32332e563db8345bef78
</code></pre>
<p>Also I’ve tried to push image from my local machine (a couple versions ago), it worked and now, when I try to inspect app-ui image with latest tag on my machine, it shows me that uploaded version lol.</p>
<p>I’m totally confused and hope it’s some kind of misunderstanding.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p><code>docker inspect</code> works only on images you have locally. Try pulling the image first and then doing <code>inspect</code>.</p>
